pyqtgraph/graphicsItems/tests/test_AxisItem.py

```diff
@@ -30,3 +30,60 @@ def test_left(p, axisSpec, tickSpecs, textSpecs):
     plot.show()
     app.processEvents()
     plot.close()
+
+
+def test_AxisItem_viewUnlink(monkeypatch):
```

Comment:
nit: I think the `monkeypatch` fixture can be removed from these tests

Reply:
Done